File: backend/src/Application/Features/Brokers/List/BrokerListItemDto.cs
using RealEstate.Domain.Enums.Brokers;

namespace RealEstate.Application.Features.Brokers.List;

public sealed record BrokerListItemDto(
    int Id,
    string FirstName,
    string LastName,
    string Email,
    string PhoneNumber,
    string? PhotoUrl,
    BrokerGender Gender,
    DateTime CreatedAt,
    DateTime UpdatedAt
);

File: frontend/src/features/brokers/api/dtos/broker-list-item.dto.ts
import type {BrokerGender} from "@/features/brokers/api/dtos/broker-details.dto.ts";

export type BrokerListItemDto = {
  id: number
  firstName: string
  lastName: string
  email?: string // on backend either phone, either email validation
  phoneNumber?: string
  photoUrl?: string
  createdAt: string
  gender?: BrokerGender
}
